src/bundler.js

```diff
@@ -71,6 +73,10 @@ export async function bundleProd({
 	cwd = cwd || '';
 	root = root || cwd;
 
+	if (!publicPath.endsWith('/')) {
+		publicPath += '/';
+	}
```

Comment:
The default `/` and trailing slash enforcement might be better in `src/lib/normalize-options.js` so that they're uniformly applied to all modes.

Edit: looks like you already have the default value there, so just the trailing. It can fall back to `/`.

Reply:
The `endsWith` check is to prevent the common webpack config issue:

```js
// wrong, but common mistake
config.publicPath = 'https://cdn.example.com';
```